providers/dev-guide.md

```diff
@@ -264,6 +264,15 @@ The logic in this class has
 quite strong assumptions on the data being stored. It assumes that it has the tree structure and each entity contains its kids as a nested hash.
 If you are able to achieve that structure in the `refresh_parser.rb`, you are halfway done. Otherwise, good luck :]
 
+### Defining Inventory
+
+Basically, refresh process consists of Collector, Persister and Parser classes (in most providers).
```

Comment:
Don't need `Basically`, and I would suggest instead of `(in most providers)` how about `Providers which use graph refresh consist of Collector ...`